### DISCUSSION TITLE
Search highlighting

### DISCUSSION THREAD

**Discussion by gixxy22** [spatie/ray#780] [Ideas]
A super handy feature would be to highlight search keywords when searching.

If I use the search bar, it will return the results with the keywords in, however if the result is an array or multidimensional array or even a longish string, it takes quite a while to actually find the keyword still.

If the search keyword could highlight the result that would be super helpful!

**Comment by freekmurze** [comment]
@Sam-Apostel Would this be easy to add?

**Comment by Sam-Apostel** [comment]
This is almost trivial to add in the items that just render text. Inserting this into all the custom highlight components is less straight forward. There are some hacky ways around that but you trade controll on which rendered values are actually used for filtering. 


Let's discuss in person later today.

**Reply by gixxy22** [reply]
Hi Sam, just wondered if you had any plans to add this feature?
